### PR TITLE
Mass hallucinations now gives the same hallucination to everyone

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
tweak: Mass Hallucination now will give everyone the same hallucination.
/:cl:

Additionally fixed a couple of runtimes and bugs in hallucination code, added an extra sound hallucinations, and buffed the mass hallucination chance a bit since according to statbus it triggered only once this month.

### CODE REVIEW

**Comment by XDTM** [comment]
Refreshing travis

File: code/modules/flufftext/Hallucination.dm [hunk 1]
```diff
@@ -157,11 +158,15 @@ GLOBAL_LIST_INIT(hallucinations_major, list(
 	cost = 25
 
 /datum/hallucination/fake_flood/New(mob/living/carbon/T, forced = TRUE)
+	set waitfor = FALSE
```
Comment (Cyberboss) [inline code/modules/flufftext/Hallucination.dm]: all you needed was /datum/hallucination, unset all these others please

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/flufftext/Hallucination.dm (hunk 1 above)
Comment (XDTM) [reply code/modules/flufftext/Hallucination.dm]: It was already set, and it didn't work on testing

Comment (Cyberboss) [reply code/modules/flufftext/Hallucination.dm]: Maybe that's a quirk of New then. Did you test that these ones work?

Comment (XDTM) [reply code/modules/flufftext/Hallucination.dm]: Yeah, intead of one waiting for each other they ran concurrently. Are you sure it's inherited? It feels like it's not meant to, especially if ..() is not used

Comment (optimumtact) [reply code/modules/flufftext/Hallucination.dm]: @Cyberboss

**Pull request merged**